src/main/java/uk/gov/ch/repository/psc/PersonsWithSignificantControlRepository.java

```diff
@@ -80,4 +80,30 @@ public interface PersonsWithSignificantControlRepository extends PagingAndSortin
                     + "AND CBA.RESIGNATION_IND = 'N' AND CBA.SUPER_SECURE_PSC_IND = 'N' AND CBA.APPOINTMENT_TYPE_ID IN (5007, 5008, 5009)",
             nativeQuery = true)
     Page<PersonWithSignificantControl> findPersonsWithSignificantControl(String companyNumber, Pageable pageable);
+
+    @Query(value = "select "
+            + "CBA.CORPORATE_BODY_APPOINTMENT_ID, "
+            + "CBA.APPOINTMENT_DATE, "
+            + "URA.STREET as RESIDENTIAL_ADDRESS_LINE_1, "
+            + "URA.AREA as RESIDENTIAL_ADDRESS_LINE_2, "
+            + "URA.CARE_OF as RESIDENTIAL_ADDRESS_CARE_OF, "
+            + "URA.COUNTRY_NAME as RESIDENTIAL_ADDRESS_COUNTRY_NAME, "
+            + "URA.POST_TOWN as RESIDENTIAL_ADDRESS_POST_TOWN, "
+            + "URA.PO_BOX as RESIDENTIAL_ADDRESS_PO_BOX, "
+            + "URA.POST_CODE as RESIDENTIAL_ADDRESS_POST_CODE, "
+            + "URA.HOUSE_NAME_NUMBER as RESIDENTIAL_ADDRESS_HOUSE_NAME_NUMBER, "
+            + "URA.REGION as RESIDENTIAL_ADDRESS_REGION, "
+            + "OD.OFFICER_DATE_OF_BIRTH, "
+            + "OD.OFFICER_NATIONALITY "
```

Comment:
Ticket mentions "second nationality/nationality other" - are we happy that's here or does there need to be another field returned?

Reply:
Second nationality is no longer needed to be fetched by this ticket as nationality that is already returned is retrieved as a comma separated list, containing the second nationality value: eg. 'BRITISH, IRISH'. The ticket has been updated to reflect this